src/Common/Enums/ROSNodeModel.cs

```diff
@@ -17,5 +17,34 @@ public class ROSNodeModel
 
         public List<ROSServiceModel> Services { get; set; }
 
+        /// <summary>
+        /// Put all node topics into a single list.
+        /// </summary>
+        /// <param name="nodes"></param>
+        /// <returns></returns>
+        public List<RosTopicModel> GetAllNodeTopics(List<ROSNodeModel> nodes)
```

Comment:
~~`GetAllNodeTopics` function should be in the RobotModel. It should take no parameters and rerurn all the topics associated with current robot instance.~~ I see that you have implemented it in RobotModel already. But still, function placed in a model should operate only on this specific instance of a Model.
Also move this class from `Enums` folder to `Models` folder

Reply:
This are different :) In robotModel there is GetAllRobotTopics() and in RosNodeModel there is GetAllNodeTopics(). One is for geting all topics of a node and the other one is for the whole robot. 
Moved script to models. Dont know why it was in Enums.

Reply:
Ok, understood. But still, I wouldn't say I like that the function named `GetAllNodeTopics` takes a `List` as a parameter. It should just return all topics in nodes without any parameters. If you want to pass a `List` as a parameter, the function should be placed in the `RobotModel` and have the name `GetTopicsForSpecifiedNodes`.